src/Classes/Listing.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+
+namespace ListingsApp\Classes;
+
+
+class Listing
+{
+ protected string $agent_ref;
+ protected string $address_1;
+ protected string $address_2;
+ protected string $town;
+ protected string $postcode;
+ protected string $description;
+ protected string $bedrooms;
+ protected string $price;
+ protected string $image;
+ protected string $type;
+ protected string $status;
+
+    /**
```

Comment:
im concerned all these docblocks are not quite descriptive enough, might need to fill in the blanks a little here